### DISCUSSION TITLE
Is there an independent emacs package that matches the functionality of the things interface?

### DISCUSSION THREAD

**Discussion by Kenneth-Mata** [meow-edit/meow#541] [Q&A]
I'm trying mostly vanilla keybindings on-off alongside meow and I'm looking for an emacs package with that bound-inner-beg-end aspect regarding different elements

**Comment by DogLooksGood** [comment]
It's trivial to roll your own. Try eval this
```
(bounds-of-thing-at-point 'line)
```

It gives you the beginning and end of a thing.
